platform/nodejs/index.js

```diff
@@ -201,6 +201,51 @@ function reset() {
     snfe.reset();
 }
 
+/******************************************************************************/
+
+let pslInitialized = false;
+let staticNetFilteringEngineInstance = null;
+
+class StaticNetFilteringEngine {
+    constructor() {
+        if ( staticNetFilteringEngineInstance !== null ) {
+            throw new Error('Only a single instance is supported.');
```

Comment:
Since there's only one `snfe` for the time being only one instance is supported. In the future it should be possible to have multiple instances.